hack/jenkins/common.sh

```diff
@@ -91,7 +91,7 @@ else
 fi
 
 # installing golang so we could do go get for gopogh
-./installers/check_install_golang.sh "1.16.4" "/usr/local" || true
+./installers/check_install_golang.sh "1.16.6" "/usr/local" || true
```

Comment:
This is still `1.16.6`, is the regex working for this file?

Reply:
good catch !